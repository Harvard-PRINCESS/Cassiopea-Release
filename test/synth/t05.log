Command: Synthesize for spec t05.spec output to t05.out; log to t05.log.raw
Output to: t05.out
Log to: t05.log.raw
Synthesis with 0 insts starting
Found dependencies in: 0.006763s
Found value dependencies in: 0.000026s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Synthesis with 0 insts failed with 1 candidates, 1 counterexamples!
Synthesis with 1 insts starting
Found dependencies in: 0.007692s
Found value dependencies in: 0.000039s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Synthesis with 1 insts failed with 1 candidates, 1 counterexamples!
Synthesis with 2 insts starting
Found dependencies in: 0.007196s
Found value dependencies in: 0.000039s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Iter 2 Checking...	Checking Time: ???s
Synthesis with 2 insts succeeded! Prog:
(EQ 0b00000000000000000000000000000000 0b00000000000000000000000000000000)
(SW 0b00000000000000000000000000000000)
	Execution Time: ???s
