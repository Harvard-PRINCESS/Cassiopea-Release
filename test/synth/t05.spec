(* github issue #159, failure case *)
let ptr: 32 bit = [allmem, 0]
frame:
  reg-modify: r
pre: true
post:
   fetch[ptr, 32] == 0x00000001

