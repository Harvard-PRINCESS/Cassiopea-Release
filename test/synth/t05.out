(EQ 0b00000000000000000000000000000000 0b00000000000000000000000000000000)
(SW 0b00000000000000000000000000000000)
